test/actor_test.py

```diff
@@ -691,11 +683,12 @@ def testActorsOnNodesWithNoCPUs(self):
 
         @ray.remote
         class Foo(object):
-            def __init__(self):
```

Comment:
Sort of off-topic, but is it okay to define an actor without an `__init__` method? I had thought that we threw an error for that.

Reply:
We *print* an error. It's valid Python and generally ok I think.